docs/ThemeColors.stories.tsx

```diff
@@ -1,8 +1,16 @@
 import React from 'react';
-import tokens from '../src/figma/tokens.json';
-import { ColorSwatchGroup } from './components';
+
+import { ColorSwatchGroup, ColorSwatch } from './components';
 import README from './ThemeColors.mdx';
 
+// Figma
+import tokens from '../src/figma/tokens.json';
+// JS
+import { lightTheme, darkTheme } from '../src';
+// CSS
+import lightThemeCss from './data/lightTheme.json';
+import darkThemeCss from './data/darkTheme.json';
+
```

Comment:
Imports all the different token data. While it is possible to read a CSS file in JavaScript using the Fetch API or XMLHttpRequest. Parsing the CSS file to extract the CSS variables can be a bit tricky. I tried that route and unfortunately got snagged